src/view/kc-change-frame.js

```diff
@@ -0,0 +1,37 @@
+import { LitElement, html } from 'lit-element';
+
+class KCChangeFrame extends LitElement {
+  static get properties () {
+    return {
+      frameTarget: { type: Number },
+      text: { type: String },
+      title: { type: String }
+    }
+  }
+
+  constructor () {
+    super();
+    this.text = '►►';
+    this.frameTarget = 1;
+    this.title = 'Next'
+  }
+
+  onClick (event) {
+    console.log(this.frameTarget);
```

Comment:
Its cool that you are getting you hand in with the creation of a component. Turely! We should have a short meeting this week to coordinate on the redux architecture and approach. I would say for example that an action "nextframe()" would be called here. This action  live in the `action/transport/nextFrame()` with other actions like play and stop. The nextFrame action would dispatch a change to the store. This way when we have keyboard short cut s and such like we use the same action everywhere. Lets talk. Its all part fo the research. There is a timeline reducer with a prop currectIndex. Try and trigger the change of the prop using an action.